### PR TITLE
VACMS 18326 content-build fontawesome removal

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Removes fontawesome and replaces with va-icon
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18326

## Testing done

- Checked manually. Forced display of some of the features locally to expose (could not find example of staff_profile without a photo)
- Check manually on tugboat (will try to create a staff profile with no image)

## Screenshots

<details>
<summary>Staff Profile without images with PR</summary>

</details>

<details>
<summary>Staff Profiles past - fontawesome</summary>
Screenshot 2024-06-15 at 4 31 48 PM
</details>

<details>
<summary>(outdated should be removed)Operation Status page - old</summary>
Screenshot 2024-06-18 at 4 10 44 PM
Screenshot 2024-06-18 at 4 10 32 PM
</details>

<details>
<summary>(outdated should be removed)Operation Status page - with PR (follows pattern from operating status page)</summary>
Screenshot 2024-06-18 at 4 10 17 PM
Screenshot 2024-06-18 at 4 10 05 PM
</details>
## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

Check that in conjunction with vets-website PR of same name that Staff profiles without photos show new va-icon and that icon is centered in gray circle. Check that Operating status page shows normal text for normal notices and slim alerts for all others.

### CODE REVIEW

**Review by chriskim2311 — APPROVED** [review]
LGTM!

File: src/site/includes/operation_status_information.liquid [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {% if fieldOperatingStatusFacility == 'notice' %}
     <span class="operating-status-flag operating-status-flag-notice operating-status usa-alert usa-alert-info background-color-only vads-u-margin-top--1 vads-u-padding-y--1 vads-u-padding-x--1p5">
-      <i class="fa fa-info-circle" aria-hidden="true"></i>
+      <va-icon icon="info" size="3" class="os-icon"></va-icon>
```
Comment (randimays) [inline src/site/includes/operation_status_information.liquid:3]: In your PR description, you indicated that this is an outdated page that should be removed. Is that something we could accomplish in this PR?

**Review by randimays — COMMENTED** [review]
Added a question.

When filling out the PR template, could you please make sure any boilerplate is removed (including the message above the Summary heading)?

File: src/site/includes/operation_status_information.liquid (hunk 1 above)
Comment (eselkin) [reply src/site/includes/operation_status_information.liquid:3]: Laura was asking that question to product people. But for know we settled on just updating it.

Comment (randimays) [reply src/site/includes/operation_status_information.liquid:3]: Ok cool!

**Review by randimays — APPROVED** [review]
LGTM

**Review by CBonade — APPROVED** [review]

**Review by jilladams — APPROVED** [review]
Approving on behalf of Facilities FE since it has FE code review and Daniel's out.

**Pull request merged**